app/views/tasks/_footer.html.erb

```diff
@@ -0,0 +1,3 @@
+<footer>
```

Comment:
I think you can delete this file since you have a footer in the shared folder, right?